test/functional/tools/realtimetool_cellxgene.xml

```diff
@@ -0,0 +1,27 @@
+<tool id="interactive_tool_cellxgene" tool_type="realtime" name="Interactive CellXgene Environment" version="0.1">
+    <requirements>
+        <container type="docker">quay.io/galaxy/cellxgene-galaxy-ie:ie2</container>
+    </requirements>
+    <realtime>
+        <entry_point name="cellxgene_single_cell_visualisation">
+            <port>80</port>
+        </entry_point>
+    </realtime>
+    <command><![CDATA[
+        cp '${input}' '/tmp/galaxy_cellxgene_${input.element_identifier}.h5ad'
```

Comment:
I'm sure you hate the best practice for dealing element_identifier Dan 😓, but please clean it the way @bgruening's tool does.

xref https://planemo.readthedocs.io/en/latest/writing_advanced.html#processing-identifiers
xref #7937

Reply:
I mean sure, I will fix this, but this was from the code that @bgruening PR'd, just cleaned up a little bit with adding quotes after the merge conflict.

Reply:
re.sub'd in 87fed7e8eede87a6ab97e1f0862a9727d9a19880

Reply:
Please treat all what I did as experimental. I just wanted to demonstrate that this kind of works for Jupyter, RStudio and where are limitations.